operator/apis/platform/v1alpha1/central_types.go

```diff
@@ -73,7 +73,7 @@ type CentralSpec struct {
 	Monitoring *GlobalMonitoring `json:"monitoring,omitempty"`
 
 	// Network configuration.
-	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=10,xDescription={"urn:alm:descriptor:com.tectonic.ui:advanced"}
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Network,order=10,xDescription={"urn:alm:descriptor:com.tectonic.ui:advanced"}
```

Comment:
This is likely why it wasn't hidden:
```suggestion
	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName=Network,order=10,xDescriptors={"urn:alm:descriptor:com.tectonic.ui:advanced"}
```

Reply:
Good catch, thank you! Will copy-paste next time